src/clj/puppetlabs/puppetserver/certificate_authority.clj

```diff
@@ -69,7 +69,18 @@
    :ruby-load-path           [schema/Str]
    :gem-path                 schema/Str
    :signeddir                schema/Str
-   :serial                   schema/Str})
+   :serial                   schema/Str
+   ;; List of for all compile master certificates including MoM
+   ;; provisioned by PE or user in case of FOSS
+   :compile-masters          [schema/Str]    
```

Comment:
still think changing our terminology from "compile-master" to "infra" or "infra-node" would be beneficial in the long run (eg. we include the MoM already).